packages/elements/src/icons/PhotoDragIcon.tsx

```diff
@@ -2,27 +2,12 @@ import * as React from "react";
-      <g id="Login-v2.1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-        <g id="08b_avatar-drop" transform="translate(-690.000000, -338.000000)" fill="#2B56FF" fill-rule="nonzero">
-          <g id="Group" transform="translate(533.000000, 130.000000)">
-            <g id="Group-5" transform="translate(78.000000, 137.000000)">
-              <g id="noun_Photo_2144503" transform="translate(79.000000, 71.000000)">
-                <path
-                  d="M49.5375494,9.4229249 L42.472332,9.4229249 L40.5454545,5.243083 C39.9130435,3.87944664 38.5395257,3 37.0375494,3 L22.3932806,3 C20.8913043,3 19.527668,3.87944664 18.8952569,5.243083 L16.958498,9.4229249 L9.89328063,9.4229249 C7.7687747,9.4229249 6,11.1521739 6,13.2766798 L6,38.0790514 C6,40.2035573 7.7687747,41.9328063 9.89328063,41.9328063 L49.5375494,41.9328063 C51.6620553,41.9328063 53.43083,40.2035573 53.43083,38.0790514 L53.43083,13.2766798 C53.43083,11.1521739 51.6620553,9.4229249 49.5375494,9.4229249 Z M51.6521739,38.0790514 C51.6521739,39.2252964 50.6837945,40.1541502 49.5375494,40.1541502 L9.89328063,40.1541502 C8.74703557,40.1541502 7.77865613,39.2252964 7.77865613,38.0790514 L7.77865613,13.2766798 C7.77865613,12.1304348 8.74703557,11.201581 9.89328063,11.201581 L17.5316206,11.201581 C17.8774704,11.201581 18.1936759,10.9940711 18.3418972,10.687747 L20.5158103,5.99407115 C20.8517787,5.26284585 21.5928854,4.78853755 22.4031621,4.78853755 L37.0573123,4.78853755 C37.8675889,4.78853755 38.5988142,5.26284585 38.944664,5.99407115 L41.1185771,10.687747 C41.2667984,11.0039526 41.583004,11.2114625 41.9288538,11.2114625 L49.5671937,11.2114625 C50.7134387,11.2114625 51.6818182,12.1403162 51.6818182,13.2865613 L51.6818182,38.0790514 L51.6521739,38.0790514 Z"
-                  id="Shape"
-                ></path>
-                <path
-                  d="M29.715415,12.5355731 C22.9268775,12.5355731 17.4031621,18.0592885 17.4031621,24.8478261 C17.4031621,31.6363636 22.9268775,37.1600791 29.715415,37.1600791 C36.5039526,37.1600791 42.027668,31.6264822 42.027668,24.8379447 C42.027668,18.0494071 36.5039526,12.5355731 29.715415,12.5355731 Z M29.715415,35.3715415 C23.9051383,35.3715415 19.1818182,30.6482213 19.1818182,24.8379447 C19.1818182,19.027668 23.9051383,14.3043478 29.715415,14.3043478 C35.5256917,14.3043478 40.2490119,19.0375494 40.2490119,24.8379447 C40.2490119,30.6482213 35.5256917,35.3715415 29.715415,35.3715415 Z"
-                  id="Shape"
-                ></path>
-                <path
-                  d="M15.2885375,15.2529644 L11.5335968,15.2529644 C11.0395257,15.2529644 10.6442688,15.6482213 10.6442688,16.1422925 C10.6442688,16.6363636 11.0395257,17.0316206 11.5335968,17.0316206 L15.2885375,17.0316206 C15.7826087,17.0316206 16.1778656,16.6363636 16.1778656,16.1422925 C16.1778656,15.6482213 15.7826087,15.2529644 15.2885375,15.2529644 Z"
-                  id="Path"
-                ></path>
-              </g>
-            </g>
```

Comment:
Removing all these nested transforming `<g>`s makes sense but a heads up that I did see that once changing the positioning of the icon and I had to adjust spacing around it. We should remove them to begin with so we only have to arrange the positioning once, cc @nickreynolds, but I think in the case of avatar upload any small translation changes will be fine, it's mainly only tricky when the icons have to be inline with text or something.

Reply:
We should ask design to optimize before sending to us